string robot_id
string file_name
string time_stamp